pkgs/applications/version-management/sourcehut/update.sh

```diff
@@ -1,8 +1,11 @@
 #! /usr/bin/env nix-shell
 #! nix-shell -i bash -p git mercurial common-updater-scripts
+set -x
```

Comment:
```suggestion
set -eoux pipefail
```

Reply:
@SuperSandro2000 I understand that we may want the others options, but why `-o`?
> If -o is supplied with no option-name, the values of the current options are printed.

Reply:
@SuperSandro2000, oh, ok I understand, it sets `pipefail`, I'll commit in a more readable way.

Reply:
Good, AFAICS setting `pipefail` does not break the `update.sh`.